File: backend/Flyt/EFGetStarted/Models/Route.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flyt.Models
{
    public class Route
    {
        public int Id { get; set; }
        public Driver Driver { get; set; }
        public Vehicle Vehicle { get; set; }
        public Stoppoint Stoppoint { get; set; }
        public DateTime Date { get; set; }
    }
}

File: frontend/src/types/Route.ts
import { Driver } from '@/types/Driver';
import { Vehicle } from '@/types/Vehicle';
import { StopPoint } from '@/types/StopPoint';

export interface Route {
  RouteId: string;
  Driver: Driver;
  Vehicle: Vehicle;
  Recipient: StopPoint;
  Date: Date;
}
